db/view/view_update_generator.cc

```diff
@@ -206,6 +207,7 @@ future<> view_update_generator::start() {
 }
 
 future<> view_update_generator::stop() {
+    _db.unplug_view_update_generator();
```

Comment:
Is it quiesced by this time?

Reply:
Should be. The v.u.g. stops after database is drained with its shutdown() call